frontend/src/components/AddClient.tsx

```diff
@@ -30,9 +33,22 @@ export default function AddClient(props: AddClientProps) {
     }
 
     return (
-        <div>
-            <input value={clientToAdd.name} onChange={handleChangeName} placeholder="name"/>
-            <button onClick={handleSubmit}>Add Client</button>
-        </div>
+            <Box
+                component="form"
+                sx={{
+                    display: 'flex',
+                    alignItems: 'center',
+                    flexDirection: 'column',
+                    '& > :not(style)': {m: 1, width: '25ch'},
+                }}
+                noValidate
+                autoComplete="off"
+                onSubmit={handleSubmit}
+            >
+                <Typography variant="h5">Neuer Kunde</Typography>
```

Comment:
This h5 looks asymmetric because its is aligned left. I suggest adding: sx={{textAlign: 'center'}